config/rbac/role.yaml

```diff
@@ -105,6 +104,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes/status
+  verbs:
+  - get
+  - patch
+  - update
```

Comment:
Hmm, why would the manager need the permission to update node status?

Reply:
hi @marquiz , there is a feature in NFD adding some labels as `resource-label`, NFD operator will support it after https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/77 merged, the `resource-label` means some 3rd party resource for the node, which is exposed in `status.allocatable` and `status.capacity` so that this permission would be needed.

Reply:
Oops, sorry for not realized you were talking about the manager. 
because the role that the operand used was created by the manager, and the manger can not grant the node/status if it did not have it.

Reply:
OK, thx for the explanation. I wasn't aware of that restriction